.travis.yml

```diff
@@ -4,7 +4,12 @@ rust:
   - stable
   - beta
   - nightly
+before_script:
+  - rustup toolchain install nightly
 script:
   - cargo build --verbose
   - cargo test --verbose
   - cargo doc
+  - cargo +nightly generate-lockfile -Z minimal-versions
+  - cargo build --verbose
+  - cargo test --verbose
```

Comment:
What do you think about just running these commands when the `nightly` matrix entry runs, instead of on every build?

Reply:
That is what I meant with my question about `configurations`. Is there some easy way to do an `if nightly` in this style of travis file?

Reply:
Oh I see what you meant now. Sorry, didn't catch that. And yeah:

```yaml
script:
  - cargo build --verbose
  - cargo test --verbose
  - cargo doc
  - if [ "$TRAVIS_RUST_VERSION" = "nightly" ]; then
      cargo +nightly generate-lockfile -Z minimal-versions;
      cargo build --verbose;
      cargo test --verbose;
    fi
```

(Note the semi-colons. This is usually when I start splitting the `script` section to a shell script, but I'm fine with this for now. I can fix that later if I care.)